sql/atxs/atxs.go

```diff
@@ -9,8 +9,8 @@ import (
 	"github.com/spacemeshos/go-spacemesh/sql"
 )
 
-// Get gets an ATX by a given ATX ID.
-func Get(db sql.Executor, id types.ATXID) (atx *types.ActivationTx, err error) {
+// ByID gets an ATX by a given ATX ID.
+func ByID(db sql.Executor, id types.ATXID) (atx *types.ActivationTx, err error) {
```

Comment:
`Get` is the consistent term in all the sql sub pkgs. i think we should either change all or leave it as Get

same for other methods. GetByXXXX

if you want change, please send a separate PR that updates all sql sub pkgs